pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system] # allows user to build pip-tools using pyproject-build or python -m build
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2", "wheel"]
```

Comment:
`wheel` is unnecessary since it's dynamically injected by setuptools anyway. I've also removed it from the examples in the setuptools' docs.
```suggestion
requires = [
  "setuptools >= 45",
  "setuptools_scm[toml] >= 6.2",
]
```

Reply:
Also, this does not actually enable setuptools-scm, the section is missing.